qutebrowser/misc/crashdialog.py

```diff
@@ -220,6 +220,12 @@ def _gather_crash_info(self):
             exc: An exception tuple (type, value, traceback)
         """
         try:
+            application = QApplication.instance()
+            launch_time = application.launch_time.strftime("%d-%m-%Y %H:%M:%S")
```

Comment:
Being European I agree with this format, but it might be confusing for others - so I changed it to `.ctime()` to use a standardized format (`Fri Aug  7 22:31:31 2015`) :wink: